awx/api/views/webhooks.py

```diff
@@ -244,3 +244,54 @@ def check_signature(self, obj):
         # analysis by attackers.
         if not hmac.compare_digest(force_bytes(obj.webhook_key), self.get_signature()):
             raise PermissionDenied
+
+
+class GiteaWebhookReceiver(WebhookReceiverBase):
+    service = 'gitea'
+
+    ref_keys = {
```

Comment:
I assume that you got these by trying it out?  I'm not finding much detail in the Gitea docs about what their webhook payloads look like, aside from a single example.

Reply:
Ah, I see a list of event types in here: https://github.com/go-gitea/gitea/blob/master/models/webhook.go

Reply:
hey @jbradberry me I can help you for more examples? What do you need? :)

Reply:
@NicoWde what is needed here is a list of event types emitted by Gitea that are tied to some commit hash, and the path in the json payload of those events under which that canonical commit hash is located.  I have no familiarity with Gitea and the documentation seems to not have much in the way of example payloads, so I don't feel like I can review this with any confidence.

It's been a while since I last looked at this PR, so I don't remember if this is the only thing that the review was stalled on.  cc @ptoal